scripts/config/rslib.config.ts

```diff
@@ -45,4 +46,14 @@ export const dualPackage = defineConfig({
       externals: commonExternals,
     },
   },
+  plugins: [
+    pluginAreTheTypesWrong({
+      areTheTypesWrongOptions: {
```

Comment:
We can only enable this on CI:
```suggestion
    pluginAreTheTypesWrong({
      enable: Boolean(process.env.CI),
      areTheTypesWrongOptions: {
```

Reply:
Will this plugin slow down the local build?

Reply:
Get. We can only enable this on CI since these kind of errors are relatively rare.